wps_shared/wps_shared/db/crud/auto_spatial_advisory.py

```diff
@@ -181,7 +181,7 @@ async def get_all_sfms_fuel_type_records(session: AsyncSession) -> List[SFMSFuel
     """
     stmt = select(SFMSFuelType)
     result = await session.execute(stmt)
-    return result.all()
+    return result.scalars().all()
```

Comment:
Returning `.scalars().all()` to avoid having tuples in the result that we have to index into.